src/components/UNSTABLE__LayoutModules/_index.scss

```diff
@@ -12,3 +12,12 @@
 @import 'DescriptionModule/index';
 @import 'ICAModule/index';
 @import 'TitleBarModule/index';
+
+@import '@carbon/themes/scss/tokens';
+
+.page-layouts__banner {
```

Comment:
Awesome! Could this be included as a Storybook style instead? For example, in `.storybook/index.scss` - this way it's not [included as part of the consumer output](https://github.com/carbon-design-system/ibm-security/pull/925/files#diff-6116f1115397b20649c09d3904780c68cc73ea5b9da45ceb0a510c33f0b34136R27186-R27191)